README.md

```diff
@@ -189,12 +189,13 @@ By default, literal matching is performed. The assertion fails if
 }
 ```
 
-The expected output can be specified with a heredoc or standard input as well.
+The expected output can be specified with a heredoc or standard input as well,
+by providing `-` as an option.
```

Comment:
Could you mention the long option in the README as well. Like we did for [other options][long-option]:

> Partial matching can be enabled with the `--partial` option (`-p` for short). When used, the assertion fails if the expected substring is not found in `$output`.

[long-option]: https://github.com/ztombol/bats-assert/blob/master/README.md#partial-matching